vertx-pg-client/src/main/asciidoc/index.adoc

```diff
@@ -114,6 +114,15 @@ A simple way to configure the client is to specify a `PgConnectOptions` data obj
 {@link examples.PgClientExamples#configureFromDataObject(io.vertx.core.Vertx)}
 ----
 
+You can also configure the generic properties with the `setProperties` or `addProperty` methods. Note `setProperties` will override the default client properties.
+
+For example, you can set a default schema for the connection with adding a `search_path` property.
```

Comment:
can you add a link to PostgreSQL connection attributes documentation to make this clearer ?

Reply:
fixed in https://github.com/eclipse-vertx/vertx-sql-client/pull/344/commits/6791157c0fd1ca24307c8eb92e12e4d3d60e2716.